modules/nw-metallb-layer2.adoc

```diff
@@ -45,6 +45,10 @@ The `speaker` pod that announces the external IP address must be on the same nod
 * Client traffic is routed to the host network and connects to the `192.168.100.200` IP address.
 After traffic enters the node, the service proxy sends the traffic to the application pod on the same node or another node according to the external traffic policy that you set for the service.
 
+** If the external traffic policy for the service is set to `cluster`, the node that advertises the `192.168.100.200` load balancer IP address is selected from the nodes where a `speaker` pod is running. Only that node can receive traffic for the service.
```

Comment:
@kquinn1204 Would it make sense o use word 'announce' instead of 'advertise' to distinguish between L2 and L3 service?

Reply:
It looks good to me otherwise,